examples/notebooks/Desbordante_demo_scenario_1_typo_miner.ipynb

```diff
@@ -0,0 +1,462 @@
+        "import desbordante\n",
+        "import pandas\n",
+        "\n",
+        "def setup_pandas_print():\n",
+        "    pandas.set_option('display.max_columns', None)\n",
+        "    pandas.set_option('display.width', None)\n",
+        "    pandas.set_option('display.max_colwidth', None)\n",
+        "    pandas.set_option('display.expand_frame_repr', False)\n",
+        "\n",
+        "setup_pandas_print()"
+      ],
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "e2ogzUWhlaWt",
+        "outputId": "0b15b82e-1187-45dc-955f-9061849dbb1f"
+      },
+      "execution_count": 1,
+      "outputs": [
+        {
+          "output_type": "stream",
+          "name": "stdout",
+          "text": [
+            "Collecting desbordante==2.3.2\n",
+            "  Downloading desbordante-2.3.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl.metadata (19 kB)\n",
+            "Downloading desbordante-2.3.2-cp311-cp311-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (4.0 MB)\n",
+            "\u001b[2K   \u001b[90m━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━\u001b[0m \u001b[32m4.0/4.0 MB\u001b[0m \u001b[31m11.6 MB/s\u001b[0m eta \u001b[36m0:00:00\u001b[0m\n",
+            "\u001b[?25hInstalling collected packages: desbordante\n",
+            "Successfully installed desbordante-2.3.2\n",
+            "--2025-03-20 17:45:38--  https://raw.githubusercontent.com/Desbordante/desbordante-core/refs/heads/main/examples/datasets/Workshop.csv\n",
+            "Resolving raw.githubusercontent.com (raw.githubusercontent.com)... 185.199.111.133, 185.199.110.133, 185.199.108.133, ...\n",
+            "Connecting to raw.githubusercontent.com (raw.githubusercontent.com)|185.199.111.133|:443... connected.\n",
+            "HTTP request sent, awaiting response... 200 OK\n",
+            "Length: 95017 (93K) [text/plain]\n",
+            "Saving to: ‘Workshop.csv’\n",
+            "\n",
+            "Workshop.csv        100%[===================>]  92.79K  --.-KB/s    in 0.009s  \n",
+            "\n",
+            "2025-03-20 17:45:39 (10.1 MB/s) - ‘Workshop.csv’ saved [95017/95017]\n",
+            "\n",
+            "Collecting colorama\n",
+            "  Downloading colorama-0.4.6-py2.py3-none-any.whl.metadata (17 kB)\n",
+            "Requirement already satisfied: jellyfish in /usr/local/lib/python3.11/dist-packages (1.1.0)\n",
+            "Downloading colorama-0.4.6-py2.py3-none-any.whl (25 kB)\n",
+            "Installing collected packages: colorama\n",
+            "Successfully installed colorama-0.4.6\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Setting up various algorithm parameters."
+      ],
+      "metadata": {
+        "id": "zRAc6mNW5T6_"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "# Value cluster filtering parameters.\n",
```

Comment:
I believe somewhere at the start we need to put text cell which will say:

This collection of scenarios demonstrates how to solve various data quality problems by exploiting patterns found (or validated) by Desbordante.

In this scenario, we showcase a simple application that performs typo detection in a table.

The idea of this scenario is described in the paper "Solving Data Quality Problems with Desbordante: a Demo" by G. Chernishev et al., available at https://arxiv.org/abs/2307.14935. There is also an interactive demo at https://desbordante.streamlit.app/.